docassemble/ALToolbox/misc.py

```diff
@@ -78,7 +80,7 @@ def fa_icon(icon: str, color="primary", color_css=None, size="sm") -> str:
         return ":" + icon + ":"  # Default to letting Docassemble handle it
     elif color_css:
         return (
-            '<i class="fa fa-'
+            f'<i class="{fa_class} fa-'
```

Comment:
Is `fa-solid` the same as `fa`? I can't actually find if it says that in the docs, though `fa-solid` is listed first in https://fontawesome.com/v6/docs/apis/javascript/import-icons#referencing-icons.

Reply:
It's the default class. In the free group there is also `fab` for brands but that's not the most common use.

Reply:
https://stackoverflow.com/questions/49913423/why-font-awesome-icon-class-is-different explains when fa was deprecated in fontawesome 5. Now there is fa-solid in fontawesome 6 which means the same thing but is clearer IMHO.